app/src/main/java/ar/com/wolox/android/example/ui/example/fragment/NewsFragment.kt

```diff
@@ -86,12 +90,34 @@ class NewsFragment @Inject constructor() : WolmoFragment<FragmentNewsBinding, Ne
         }
     }
 
-    override fun setDataNews(item : News){}
+    override fun setDataNews(item : News, userid : Int){}
 
     override fun adapterRefresh(){
         adapter.notifyDataSetChanged()
     }
 
+    override fun onItemClick(new : News) {
+        NewDetailActivity.start(requireContext(),new)
+    }
+
+    //Cuando se resume el fragment, se chequea si se hicieron cambios para refrescar las noticias
+    override fun onResume(){
+        super.onResume()
+
+        //Verifico si algo cambio de la noticia para refrescar
+        val pref = context?.getSharedPreferences(BaseConfiguration.SHARED_PREFERENCES_NAME, Context.MODE_PRIVATE)
+        if(pref!=null && pref.getBoolean(Extras.Constantes.NEW_CHANGE,false)) {
+            //Limpio la lista
+            presenter.clearData()
+
+            //Obtengo nuevamente las noticias
+            presenter.getNews(false)
+
+            //Guardo que no hay cambios para que no se refresque siempre
+            pref.edit().putBoolean(Extras.Constantes.NEW_CHANGE,false).apply()
```

Comment:
Remove this line, you shouldn't write o read Shared Preferences data from the view.